example/src/main/scala/example/HelloWorldWithMiddlewares.scala

```diff
@@ -14,12 +15,15 @@ import java.util.concurrent.TimeUnit
 
 object HelloWorldWithMiddlewares extends App {
 
+  // The content type header will be added based on the extension of the requested file by smartContentType
+  val static = Method.GET / "static" / *[String] to { req => Response.text(req.params) }
+
   val app: HttpApp[Clock, Nothing] = HttpApp.collectM {
     // this will return result instantly
     case Method.GET -> !! / "text"         => ZIO.succeed(Response.text("Hello World!"))
     // this will return result after 5 seconds, so with 3 seconds timeout it will fail
     case Method.GET -> !! / "long-running" => ZIO.succeed(Response.text("Hello World!")).delay(5 seconds)
-  }
+  } +++ static
```

Comment:
It allowed me to easily test with multiple values for content type, but I agree it might be overkill for this example. I'll change it to a simple route